### PR TITLE
WIP: Add new endpoints for viewing the receipt page or an error page

### PR DESCRIPTION
**Pull request by dianakhuang**
on return from CyberSource.

@stephensanchez Here's what I have so far. We can talk it out some more later.

### CODE REVIEW

**Comment by dianakhuang** [comment]
@rlucioni Here's the PR of the work I've been doing.

File: extensions/payment/constants.py [hunk 1]
```diff
@@ -47,3 +53,88 @@ class CybersourceConstants(object):
     DECLINE = u'DECLINE'
     ACCEPT = u'ACCEPT'
     ERROR = u'ERROR'
+
+
+class CybersourceErrors(object):
+    """ The mapping of Cybersource error codes to their meanings. """
+    REASONCODE_MAP = defaultdict(lambda: "UNKNOWN REASON")
+    REASONCODE_MAP.update(
```
Comment (clintonb) [inline extensions/payment/constants.py]: Do these really need to be translated? Are they being displayed anywhere?

Comment (dianakhuang) [inline extensions/payment/constants.py]: I imagine that they'll be on the error pages we show to students eventually unless it makes more sense for us to just show them the error codes.

**Comment by rlucioni** [comment]
Thanks @dianakhuang. I'll close this now and open a new PR once this is completed.

**Pull request closed without merging**